modules/distribution-workflowoperation/src/main/java/org/opencastproject/workflow/handler/distribution/ConfigurablePublishWorkflowOperationHandler.java

```diff
@@ -124,6 +136,36 @@ protected DownloadDistributionService getDistributionService() {
     return distributionService;
   }
 
+  private void addMetadataCatalogsToTemplateData(Map<String, Object> values, MediaPackage mp) {
+    if (! mp.hasCatalogs()) {
+      logger.info("No metdata catalogs found.");
+      return;
+    }
+    if (values == null) values = new HashMap<>();
```

Comment:
Since `values` is not returned by this method, in case you create a new Map here, all data you add to it will be lost at the end of the method.